gazebo/content.md

```diff
@@ -57,12 +57,12 @@ If you not only wish to run Gazebo, but develop for it too, i.e. compile custom
 
 ## Deployment example
 
-In this short example, we'll spin up a new container running gazebo server, connect to it using a local gazebo client, then spawn a double inverted pendulum and record the simulation for later playback.
+In this short example, we'll spin up a new container running gazebo server, connect to it using a local gazebo client, then spawn a double inverted pendulum and record the simulation for later playback. To ensure compatible Gazebo versions, use the latest Gazebo version provided by the OSRF repos, not from the Ubuntu Universe.
```

Comment:
Wait, this is on the Docker image, which already uses the OSRF repos; what's the intent behind this addition?